privacyidea/lib/token.py

```diff
@@ -2111,6 +2111,7 @@ def create_challenges_from_tokens(token_list, reply_dict, options=None):
                     challenge_info["password_change"] = \
                         token_obj.is_pin_change(
                             password=True)
+                # FIXME: This is deprecated and should be remove one day
```

Comment:
Before removing this we **must** make sure that all plugins support/use the ``multi_challenge`` entry.

Reply:
Totally agree.

Reply:
i just wanted some kind of reminder here :-)